src/pages/docs/octopus-cloud/getting-started-with-cloud.mdx

```diff
@@ -23,78 +23,90 @@ After you create an Octopus account, you can create a new instance of Octopus Cl
 
 To create a new instance of Octopus Cloud, make sure you’re logged in to your Octopus account:
 
-1. Click **Create a new cloud instance**.
-2. Enter the URL you’d like to use to access your instance: `your-url.octopus.app` . If the URL isn’t available, enter a different URL.
+1. Click **New Cloud Instance**.
+2. Enter the URL you’d like to use to access your instance: `your-url.octopus.app`. If the URL isn’t available, enter a different URL.
 3. Select the region where you’d like to host your instance.
 4. Add your company name.
 5. Review the terms of our customer agreement, privacy policy, and acceptable usage policy.
-6. Click **Agree, deploy my Octopus**!
+6. Click **Start your trial**.
 
 You’ll be taken to the account provisioning screen. Your Octopus Cloud instance should be ready in a minute or so. You’ll receive an email when the instance is ready to use.
 
-When the instance is ready, you’ll see it (and any other instances you have access to) the next time you log in to your Octopus account at [octopus.com/signin](https://octopus.com/signin).
+## Managing Octopus Cloud subscriptions \{#managing-octopus-cloud-subscriptions}
 
-## Uploading packages to Octopus Cloud servers \{#uploading-packages-to-octopus-cloud-servers}
+:::div{.hint}
+We are in transition between two systems for managing Octopus Cloud subscriptions. We refer to the new system as Control Center V2, and it replaces the original [Control Center V1](https://octopus.com/control-center). **Accounts** are created in both systems but all new **subscriptions** are created in [Control Center V2](https://billing.octopus.com/).
+:::
 
-Factors like geographical distance, network bandwidth, and network congestion may cause package uploads to your Octopus Cloud Server to take longer than expected.
+You can find all of your Octopus Cloud subscriptions in [Control Center V2](https://billing.octopus.com/).
 
-If you’re having difficulty uploading packages to your Octopus built-in package feed in the default timeout threshold, typically within 5 minutes, you might be affected by one or more of the factors mentioned above. In this case, you can try increasing the timeout threshold of the Octopus CLI or nuget.exe to a value that suits you. We also recommend using the Octopus CLI, **Octopus.Client**, or the **TeamCity Plugin (v4.41.0+)** because they use delta compression.
-
-## Change your password \{#change-your-password}
```

Comment:
Moved this down the page as I think it is more important to guide users how to find their instance before worrying about how they should change their password.